src/components/modal/modal.js

```diff
@@ -361,6 +361,11 @@ export default Vue.extend({
     }
   },
   methods: {
+    updateModel(val) {
+      if (val !== !!this.visible) {
```

Comment:
Why the `!!`? The `visible` prop should always be a boolean.

Reply:
"Should" be is the keyword, except when someone uses a non Boolean on the prop (and has dev mode off which won't warn the user)

Reply:
The other option was to use:
```js
if (val === !this.visible)
```

Reply:
Hmm, but we expect the type at most other places... But I'am also fine with using it.

Reply:
I suppose we should probably emit it regardless of type... that way the model is updated correctly